### PR TITLE
experimental(terraform): Initial Conventional Terraform

### PR DESCRIPTION
**Pull request by rogerthatdev**
- Reorganizes ops and trigger terraform resources into an individual ops module.
- Refactors how services are enabled via terraform by adding a toggle variable
- Boiler plate instructions for deploying the ops environment

### CODE REVIEW

File: experimental/conventional-model/terraform/modules/ops/services.tf [hunk 1]
```diff
@@ -0,0 +1,28 @@
+locals {
+  services = var.enable_apis ? [
```
Comment (grayside) [inline experimental/conventional-model/terraform/modules/ops/services.tf:2]: I like the simplification of a list of services to be enabled, but why gate this behind a var?

File: experimental/conventional-model/terraform/modules/ops/triggers.tf [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+resource "google_cloudbuild_trigger" "api_unit_tests_build_trigger" {
+  project        = var.project_id
+  count          = var.deploy_triggers ? 1 : 0
+  name           = "api-unit-tests"
+  filename       = "ops/unit-tests.cloudbuild.yaml"
+  included_files = ["content-api/**"]
+  substitutions = {
+    _DIR             = "content-api"
+    _SERVICE_ACCOUNT = "restricted-test-identity@emblem-ops.iam.gserviceaccount.com"
```
Comment (grayside) [inline experimental/conventional-model/terraform/modules/ops/triggers.tf:16]: Is this not managed in Terraform? I don't see it elsewhere in this PR.

**Review by grayside — APPROVED** [review]

**Comment by grayside** [comment]
Thanks for your first PR! This is looking like a great first iteration for experimental terraform refactor. Thanks for putting this together.

File: experimental/conventional-model/terraform/modules/ops/main.tf [hunk 3]
```diff
@@ -0,0 +1,116 @@
+data "google_project" "target_project" {
+  project_id = var.project_id
+}
+
+# Create this topic to emit writes to Artifact Registry as events.
```
Comment (ace-n) [inline experimental/conventional-model/terraform/modules/ops/main.tf:5]:
```suggestion
# Create this topic to emit write notifications from Artifact Registry as events.

**Review by ace-n — APPROVED** [review]
**Note:** I have ignored formatting/grammar nits here.

File: experimental/conventional-model/terraform/modules/ops/services.tf (hunk 1 above)
Comment (rogerthatdev) [reply experimental/conventional-model/terraform/modules/ops/services.tf:2]: gated behind a var as suggested by best practices.  defaulted to true but gives the option to relieve Terraform from managing services.

File: experimental/conventional-model/terraform/modules/ops/triggers.tf (hunk 2 above)
Comment (rogerthatdev) [reply experimental/conventional-model/terraform/modules/ops/triggers.tf:16]: Totally copied as is from the original [cloud_build_triggers.tf](https://github.com/GoogleCloudPlatform/emblem/blob/f0553b15786bc00efb400e1abe5923171ce6586a/terraform/ops/triggers/cloud_build_triggers.tf#L15) 

will change to a dynamic project id

File: experimental/conventional-model/terraform/modules/ops/variables.tf [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+variable "repo_owner" {
+  description = "Repo Owner (GoogleCloudPlatform)"
+  type        = string
+  default     = ""
+}
+
+variable "repo_name" {
+  description = "Repo Name (emblem)"
+  type        = string
+  default     = ""
```
[the remark this answers is not in the document]
Comment (rogerthatdev) [reply experimental/conventional-model/terraform/modules/ops/variables.tf:21]: Left default value "" to allow it to execute with the deploy_triggers toggle set to false without having to provide a value.

[the remark this answers is not in the document]
Comment (ace-n) [reply experimental/conventional-model/terraform/modules/ops/variables.tf:21]: I'm not sure I understand. Are you saying that specifying a value here breaks Terraform if `deploy_triggers` is set to `false`?

**Pull request merged**